.github/workflows/test-query-engine.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       single_threaded: true
 
   postgres-push:
-    if: github.event_name == 'push'
+#    if: github.event_name == 'push'
```

Comment:
Enabled testing all databases on PR validation for complete testing. These lines will need to be uncommented before merging this PR.